src/lib/utils.js

```diff
@@ -0,0 +1,49 @@
+import { MissionUtils } from '@woowacourse/mission-utils';
+import {
+  ERROR_MESSAGE_NOT_INTEGER,
+  ERROR_MESSAGE_NOT_NUMBER,
+  ERROR_MESSAGE_NOT_POSITIVE_POSITIVE,
+} from './constants.js';
+
+export const readLineAsync = async message =>
+  MissionUtils.Console.readLineAsync(message);
+
+export const print = message => {
+  MissionUtils.Console.print(message);
+};
+
+export const pickNumberInRange = (start, end) =>
+  MissionUtils.Random.pickNumberInRange(start, end);
+
+export const splitIntoArray = (str, separator) => str.split(separator);
+
+export const getIsNumeric = num => {
+  if (typeof num !== 'number') return false;
+  return !Number.isNaN(num) && !Number.isNaN(parseFloat(num));
+};
```

Comment:
```suggestion
export const getIsNumeric = num => {
  return typeof num === 'number' || !Number.isNaN(Number(num));
};
```

여기서 `parseFloat`을 사용하신 이유와 `isNaN`을 두번 사용하신 이유가 궁금합니다! `typeof num === number`도 같은 동작을 하지 않나요?

Reply:
`parseFloat`는 `[]`같은 빈 배열을 따로 처리해주려고 했습니다. 
그런데, `getIsNumeric`을 사용하는 측에서 이미 +로 숫자화를 하고 있고, 위에서 typeof `number`를 처리하기에 이는 필요가 없군요.

`isNaN`을 두 번 사용할 필요도 없는 거 같습니다. 다시 한 번 생각해보게 되네요. 감사합니다.